Vagrantfile

```diff
@@ -0,0 +1,20 @@
+Vagrant.configure('2') do |config|
+  config.ssh.forward_agent = true
+  config.vm.box = 'cargomedia/debian-7-amd64-default'
+
+  config.vm.hostname = 'janus-gateway-js.dev.cargomedia.ch'
+
+  config.vm.network :private_network, ip: '10.10.10.16'
+  config.vm.synced_folder '.', '/home/vagrant/janus-gateway-js', :type => 'nfs'
```

Comment:
No need for `:type => 'nfs'` here, it actually complicates things (asks for password when vagrantupping)